src/screens/Home/OnboardingScreen/stepStyles.tsx

```diff
@@ -21,70 +20,67 @@ const styles = StyleSheet.create({
     alignItems: 'center',
   },
 
-  header: {
+  title: {
     color: '#DEA3BB',
     fontFamily: 'montserrat-regular', 
     fontWeight: 'bold',
     fontSize: 30,
     marginBottom: '10%',
+    marginTop: '15%'
   },
 
-  stepOne: {
+  textOne: {
     fontFamily: 'montserrat-regular', 
     fontStyle: 'normal',
     fontWeight: 'bold',
     fontSize: 30,
     lineHeight: 37,
     textAlign: 'center',
     color: '#F5F5F5',
```

Comment:
Lines 33-39 could be the generic style 'text' which could be applied to all the steps then textOne, two, three, four would just have marginBottom, height and width